api/src/controllers/AuthenticationController.ts

```diff
@@ -1,56 +1,58 @@
 import { Request, Response } from "express";
-import AuthService from "../services/AuthService";
-import ErrorController from "./ErrorController";
 import { UserAttributes } from "../utils/types/attributeTypes";
-import {
-  generateSignedJwt,
-  hashPassword,
-  verifyPassword,
-} from "../utils/tokenService";
+import AuthService from "../services/AuthenticationService";
 
-export default class AuthController extends ErrorController {
+type GenericClaimStructure = {
+  iss: string;
+  sub: string;
+  aud: string;
+  exp: number;
+  iat: number;
+};
+
+interface UserLogin {
+  username: string;
+  password: string;
+}
+
+export default class AuthenticationContoller {
   private authService = new AuthService();
 
-  public async signin(req: Request, res: Response) {
+  public async signIn(req: Request, res: Response) {
     try {
-      const data: UserAttributes = req.body;
-
+      const data: UserLogin = req.body;
```

Comment:
S: Validate format of `req.body`, before annotating type from `any`.

Reply:
Going to fix in the Error Handling PR as need to think on it a bit more.